python/whylogs/core/feature_weights.py

```diff
@@ -0,0 +1,76 @@
+import json
+from typing import Any, Dict, Tuple, TypedDict
+
+from whylogs.api.writer import Writers
+from whylogs.api.writer.writer import Optional, Writable, Writer
+
+
+class FeatureWeightResponse(TypedDict):
+    segmentWeights: list
+    metadata: Dict
+
+
+class FeatureWeightDict(TypedDict):
+    segment: Optional[str]
+    weights: Dict[str, float]
```

Comment:
In a core class, I don't think we should have separate classes for response vs the conceptual object. What about having a single class be named `FeatureWeights`?